base/string.jl

```diff
@@ -2,7 +2,11 @@
 
 print(io::IO, x) = show(io, x)
 print(io::IO, xs...) = for x in xs print(io, x) end
```

Comment:
To cover all cases of printing we would need to have it in all methods of `print`, `show`, `dump`, `xdump`, etc. Assuming that `println` would be the typical case, I added it only there.